docs/ExtensionPoints/ProductSubscription/README.md

```diff
@@ -131,7 +131,6 @@ extend(ExtensionPoint.SubscriptionManagementAdd, (root, api) => {
     onAction: () => {
       console.log('Added');
       done();
```

Comment:
Same comment as the other PR, the description for `done` needs to say that the container will be closed automatically